common/app/views/fragments/page/head/checkModuleSupport.scala.html

```diff
@@ -0,0 +1,8 @@
+@*
+Check for module support so that we can polyfill appropriately
```

Comment:
Something I double checked:

```suggestion
Check for module support so that we can polyfill appropriately
nomodule support is exactly the same as type="module"
https://caniuse.com/#feat=es6-module
```